addons/website_event_track/static/src/interactions/event_track_reminder.js

```diff
@@ -0,0 +1,78 @@
+import { _t } from "@web/core/l10n/translation";
+import { rpc } from "@web/core/network/rpc";
+import { Component } from "@odoo/owl";
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+export class WebsiteEventTrackReminder extends Interaction {
+    static selector = ".o_wetrack_js_reminder";
+
+    dynamicContent = {
+        _root: { "t-on-click.prevent.stop.withTarget": this.debounced(this.onReminderToggleClick, 500, true) },
+    };
+
+    async onReminderToggleClick(ev, currentTargetEl) {
+        const trackLinkEl = currentTargetEl.querySelector("i");
+
+        if (this.reminderOn === undefined) {
+            this.reminderOn = trackLinkEl.dataset.reminderOn;
+        }
+
+        const reminderOnValue = !this.reminderOn;
+
+        const result = await this.waitFor(rpc("/event/track/toggle_reminder", {
+            track_id: parseInt(trackLinkEl.dataset.trackId),
+            set_reminder_on: reminderOnValue,
+        }));
+
+        if (result.error && result.error === "ignored") {
+            this.services.notification.add(
+                _t("Talk already in your Favorites"),
+                {
+                    type: "info",
+                    title: _t("Error"),
+                }
+            );
+        } else {
+            this.reminderOn = reminderOnValue;
+            this.updateDisplay(trackLinkEl);
+            const message = this.reminderOn
+                ? _t("Talk added to your Favorites")
+                : _t("Talk removed from your Favorites");
+            this.services.notification.add(message, {
+                type: "info",
+            });
+            if (this.reminderOn) {
+                Component.env.bus.trigger("open_notification_request", [
+                    "add_track_to_favorite",
+                    {
+                        title: _t("Allow push notifications?"),
+                        body: _t(
+                            "You have to enable push notifications to get reminders for your favorite tracks."
+                        ),
+                        delay: 0,
+                    },
+                ]);
+            }
+        }
+    }
+
+    updateDisplay(bellSelectorEl) {
+        if (this.reminderOn) {
+            bellSelectorEl.classList.add("fa-bell");
+            bellSelectorEl.classList.remove("fa-bell-o");
+            bellSelectorEl.setAttribute("title", _t("Favorite On"));
+        } else {
+            bellSelectorEl.classList.add("fa-bell-o");
+            bellSelectorEl.classList.remove("fa-bell");
+            bellSelectorEl.setAttribute("title", _t("Set Favorite"));
+        }
```

Comment:
What about replacing this method by a `t-att-class` and a `t-att-title` on `i` ?
The `updateContent` is supposed to be implicitly called after the event handler returns.